stl/CMakeLists.txt

```diff
@@ -392,11 +392,16 @@ set(SOURCES_SATELLITE_CODECVT_IDS
     ${CMAKE_CURRENT_LIST_DIR}/src/ulocale.cpp
 )
 
+set(SOURCES_SATELLITE_ATOMIC_REF

```

Comment:
I want to merge this stuff with the atomic_wait satellite after that lands. No change requested right now, I'll take care of it.